### ISSUE TITLE
OpenViduReact at version 2.7.1 has a import problem.

### ISSUE THREAD

**Issue by keisen** [OpenVidu/openvidu-call-react#6]
Hi @pabloFuente . I'm facing a problem now.

```
./node_modules/openvidu-react/dist/components/VideoRoomComponent.js
Module not found: Can't resolve '/home/pablo/Desktop/openvidu-call-react/library/node_modules/@babel/runtime/helpers/esm/assertThisInitialized' in '/home/vagrant/workspace/openvidu-tutorials/openvidu-library-react/node_modules/openvidu-react/dist/components'
```

I checked out current (today's) master branch of `openvidu-tutorials` and I performed `npm install` in `./openvidu-tutorials/openvidu-library-react` .
The head of `./node_modules/openvidu-react/dist/components/VideoRoomComponent.js` was below:

```
import _classCallCheck from "/home/pablo/Desktop/openvidu-call-react/library/node_modules/@babel/runtime/helpers/esm/classCallCheck";
import _createClass from "/home/pablo/Desktop/openvidu-call-react/library/node_modules/@babel/runtime/helpers/esm/createClass";
import _possibleConstructorReturn from "/home/pablo/Desktop/openvidu-call-react/library/node_modules/@babel/runtime/helpers/esm/possibleConstructorReturn";
import _getPrototypeOf from "/home/pablo/Desktop/openvidu-call-react/library/node_modules/@babel/runtime/helpers/esm/getPrototypeOf";
import _inherits from "/home/pablo/Desktop/openvidu-call-react/library/node_modules/@babel/runtime/helpers/esm/inherits";
import _assertThisInitialized from "/home/pablo/Desktop/openvidu-call-react/library/node_modules/@babel/runtime/helpers/esm/assertThisInitialized";
```

And then, It's working well when I downgraded `openvidu-react` from `2.7.1` to `2.6.0`.
The latest version seems that have a problem.
I hope this report will be of some help.

Best Regards,
Keisen.

_Originally posted by @keisen in https://github.com/OpenVidu/openvidu-call-react/issues/5#issuecomment-454697987_

**Comment by pabloFuente** [comment]
Okey, so there was a dependency update related to react libraries that broke the building of openvidu-react package. Roll back to previous versions of react dependencies has been done.

New version 2.7.2 has been released to NPM and 2.7.1 has been deprecated. We have also updated openvidu-library-react tutorial to use this new version. It should work fine now.

Please check it out and post here your results.

Tnx!

**Comment by keisen** [comment]
Thank you for your quick response.
I made sure that the latest version is working and having no problem. Great!

All the Best!
